test/ospy/data2/ut_counties.prj

```diff
@@ -1 +0,0 @@
-GEOGCS["GCS_North_American_1983",DATUM["D_North_American_1983",SPHEROID["GRS_1980",6378137,298.257222101]],PRIMEM["Greenwich",0],UNIT["Degree",0.0174532925199433]]
```

Comment:
No blank line at the end! It's good that we delete this. 😉

Reply:
is it in the spec to have a blank line at the end? i didn't know that

Reply:
Not in the prj spec. It' just that many old tools assume textfiles end with a newline. That's why Github actively a symbol for it.